CMakeLists.txt

```diff
@@ -332,12 +333,22 @@ add_executable(SeisSol-bin src/main.cpp)
 target_link_libraries(SeisSol-bin PUBLIC SeisSol-lib)
 set_target_properties(SeisSol-bin PROPERTIES OUTPUT_NAME "SeisSol_${EXE_NAME_PREFIX}${PLASTICITY_NAME_SUFFIX}")
 
-add_executable(SeisSol-proxy
-        auto_tuning/proxy/src/proxy_seissol.cpp
-        auto_tuning/proxy/src/flop_counter.cpp
-)
-target_link_libraries(SeisSol-proxy PUBLIC SeisSol-lib)
-set_target_properties(SeisSol-proxy PROPERTIES OUTPUT_NAME "SeisSol_proxy_${EXE_NAME_PREFIX}")
+# SeisSol proxy-core
+add_library(proxy-core auto_tuning/proxy/src/proxy_seissol.cpp
```

Comment:
Could we please keep using the prefix SeisSol- for all targets? Makes it a bit more organized.

Reply:
sure. Btw, is it necessary to have `SeisSol` as a prefix? I mean why can we just use prefix `seissol` (it is just easy to type)

Reply:
done

Reply:
In principle it doesnt matter but wouldn't it be annoying to change it now? Imo it's fine as long as it is consistent